example/App.vue

```diff
@@ -1,9 +1,9 @@
 <template>
   <div>
     <div id="app">
-      <VueElementLoading :active="fullscreenGreenBg" is-full-screen backgroundColor="rgba(66, 185, 131, .9)" />
-      <VueElementLoading :active="fullscreen" is-full-screen/>
-      <VueElementLoading :active="show" spinner="bar-fade-scale" color="#FF6700"/>
+      <VueElementLoading :active="fullscreenGreenBg" :spinner="spinnerKind" :size="spinnerSize" :duration="spinnerDuration" is-full-screen backgroundColor="rgba(66, 185, 131, .9)"/>
+      <VueElementLoading :active="fullscreen" :spinner="spinnerKind" :size="spinnerSize" :duration="spinnerDuration" is-full-screen color="#0088FF"/>
+      <VueElementLoading :active="show && !fullscreen && !fullscreenGreenBg" :spinner="spinnerKind" :size="spinnerSize" :duration="spinnerDuration" color="#FF6700"/>
```

Comment:
Can you add props `color` to the example like `duration` and `size`?

like this
```html
<VueElementLoading :active="fullscreenGreenBg" :spinner="spinnerKind" :size="spinnerSize" :duration="spinnerDuration" is-full-screen backgroundColor="rgba(66, 185, 131, .9)" :color="spinnerColor"/>
<VueElementLoading :active="fullscreen" :spinner="spinnerKind" :size="spinnerSize" :duration="spinnerDuration" is-full-screen :color="spinnerColor"/>
<VueElementLoading :active="show && !fullscreen && !fullscreenGreenBg" :spinner="spinnerKind" :size="spinnerSize" :duration="spinnerDuration" :color="spinnerColor"/>
```